### DISCUSSION TITLE
Stuck Roll in cfclient

### DISCUSSION THREAD

**Discussion by FallowLand** [bitcraze/discussions#198] [Q&A] [answered]
Hi Bitcraze,

I'm currently using windows and I've installed VM. My problem is, with my cfclient running in windows, my roll is always stuck on 30 deg/sec. In VM, the roll is fine but my yaw is -200 deg/sec. How can I set these to zero?

**Comment by krichardsson** [comment, marked as answer]
Hi, 
it sounds like this problem makes it hard to fly!

I'm not sure why this happens, [but I would try to re-map your game pad](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#creating-new-mappings).
